### PR TITLE
feat: remove redundant naming in buttons

### LINKED ISSUE #4399: Buttons have redundant naming (Prune images, Create an image), should be just Prune and Create.
### Is your enhancement related to a problem? Please describe

Throughout the listing, the names are redundant. You click on the Image section, you expect a "Create" button.

But instead we have Create an Image, Prune Images, etc.

It should be implied that if you click on "Prune" in the Images section, it's related to images.

Screenshot 2023-10-18 at 8 58 13 AM

### Describe the solution you'd like

Should be Prune, Build, Create.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by edvardsanta**
- adjusted one test
Signed-off-by: edvardsanta <eduardosantamarco123@outlook.com>

### What does this PR do?
Remove redundant naming of the sections, for example, if you are in Volume section, implicitly Create button is to Create a new Volume, etc..

### Screenshot/screencast of this PR
image
image
image
image

### What issues does this PR fix or reference?
Closes: #4399 

### How to test this PR?
See all sections to check redudant namings of the buttons

### CODE REVIEW

**Comment by deboer-tim** [comment]
Change looks good, but please see failing smoke e2e tests, there are some tests that need to be updated with the new button names.

**Comment by edvardsanta** [comment]
> Change looks good, but please see failing smoke e2e tests, there are some tests that need to be updated with the new button names.

i will do it

**Comment by mairin** [comment]
screenshots look good @edvardsanta - thank you for this work! 🙇🏻‍♀️

**Review by jeffmaury — CHANGES_REQUESTED** [review]
Update smoke tests

**Comment by slemeur** [comment]
Thanks for the PR @edvardsanta !

File: packages/renderer/src/lib/engine/Prune.svelte [hunk 1]
```diff
@@ -74,6 +74,4 @@ async function prune(type: string) {
 }
 </script>
 
-<Button on:click="{() => openPruneDialog()}" title="Remove unused images" icon="{faTrash}">
-  Prune {type}
-</Button>
+<Button on:click="{() => openPruneDialog()}" title="Remove unused images" icon="{faTrash}">Prune</Button>
```
Comment (deboer-tim) [inline packages/renderer/src/lib/engine/Prune.svelte]: Looks like we had an existing bug! Could you fix the tooltip here too: `title="Remove unused {type}"`?

File: packages/renderer/src/lib/ContainerList.svelte [hunk 2]
```diff
@@ -418,7 +418,7 @@ function errorCallback(container: ContainerInfoUI, errorMessage: string): void {
     {#if $containersInfos.length > 0}
       <Prune type="containers" engines="{enginesList}" />
     {/if}
-    <Button on:click="{() => toggleCreateContainer()}" icon="{faPlusCircle}">Create a container</Button>
+    <Button on:click="{() => toggleCreateContainer()}" icon="{faPlusCircle}">Create</Button>
```
Comment (deboer-tim) [inline packages/renderer/src/lib/ContainerList.svelte]: Would you mind adding a tooltip via `title="Create a container"`?

File: packages/renderer/src/lib/volume/VolumesList.svelte [hunk 3]
```diff
@@ -190,7 +190,7 @@ function gotoCreateVolume(): void {
 <NavPage bind:searchTerm="{searchTerm}" title="volumes">
   <div slot="additional-actions" class="space-x-2 flex flex-nowrap">
     {#if providerConnections.length > 0}
-      <Button on:click="{() => gotoCreateVolume()}" icon="{faPlusCircle}">Create a volume</Button>
+      <Button on:click="{() => gotoCreateVolume()}" icon="{faPlusCircle}">Create</Button>
```
Comment (deboer-tim) [inline packages/renderer/src/lib/volume/VolumesList.svelte]: Could you add a tooltip `title="Create a volume"`?

File: packages/renderer/src/lib/engine/Prune.svelte (hunk 1 above)
Comment (edvardsanta) [reply packages/renderer/src/lib/engine/Prune.svelte]: Solved

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by lstocchi — APPROVED** [review]

**Comment by deboer-tim** [comment]
e2e smoke test failure looks unrelated (possibly network issue) so I reran it. Thank you @edvardsanta, this looks great!

**Comment by edvardsanta** [comment]
> e2e smoke test failure looks unrelated (possibly network issue) so I reran it. Thank you @edvardsanta, this looks great!

Good. Thank you for the feedback

**Comment by deboer-tim** [comment]
Thanks @edvardsanta !

**Pull request merged**